src/openrct2-ui/windows/Guest.cpp

```diff
@@ -687,10 +687,11 @@ void window_guest_overview_viewport_rotate(rct_window *w)
 void window_guest_viewport_init(rct_window* w){
     if (w->page != WINDOW_GUEST_OVERVIEW) return;
 
-    union{
+    struct
```

Comment:
If this is not a `union`, I don't see a reason for it to be a `struct`. Please split it to separate variables.

Reply:
Done.